services/API-service/src/api/point-data/point-data.service.ts

```diff
@@ -219,6 +219,7 @@ export class PointDataService {
 
       const validationError = await validate(pointDto);
       if (validationError.length > 0) {
+        console.log('Seed point data validation error: ', validationError);
```

Comment:
Use `this.logger.log` instead of `console.log`. See [SeedAdminAreaData](https://github.com/rodekruis/IBF-system/blob/32a7ac8a02f0794112b4ec588258a525331c5c72/services/API-service/src/scripts/seed-admin-area-data.ts#L21) for reference.